builtin/credential/approle/path_role_test.go

```diff
@@ -1121,14 +1121,491 @@ func TestAppRole_RoleCRUD(t *testing.T) {
 	var err error
 	b, storage := createBackendWithStorage(t)
 
+	roleData := map[string]interface{}{
```

Comment:
Unless I am reading this diff wrong, the tests are for RUD of all the other fields but the fields that are getting modified in this PR. I assume that you wanted to the that immediately follows this to do the testing of the new fields. But that has test steps for all the other fields as well. Am I missing something?

Reply:
Maybe? This test is identical to the RUD one above, but instead of looking for "bound_cidr_list", it covers "secret_id_bound_cidrs" at the role level. I wanted to make sure the previous test worked, and that the new field worked in an identical fashion. You're right though that I didn't add update tests for the new ones, and I should add that.